lab_4_retrieval_w_clustering/main.py

```diff
@@ -129,6 +187,7 @@ def get_tokenizer(self) -> Tokenizer:
         Returns:
             Tokenizer: Tokenizer class object.
         """
+        return self._tokenizer
```

Comment:
so, here it is correct, why in vectorizer func you return `new_object`?

Reply:
following the annotations, i would like to mention `new` object))) but i agree it is strange